### ISSUE TITLE
Resize/Quality Algorithm

### ISSUE THREAD

**Issue by leeferfeefer** [bamlab/react-native-image-resizer#242]
Hello, what algorithm is this library using to reduce the quality of images?

**Comment by andreydos** [comment]
The bad algorithm as I think) I just compress 1.1MB jpg image with quality=50 and get a 1.3MB jpg output file, the quality=30 give me 0.7MB, not a big problem =))

**Comment by ggunti** [comment]
@andreydos yes, I also observed that resized image size could be bigger than original image size
